tests/common/query_context_generator.py

```diff
@@ -199,16 +199,16 @@ def get_query_object(
     query_name: str, add_postprocessing_operations: bool, add_time_offsets: bool,
 ) -> Dict[str, Any]:
     if query_name not in QUERY_OBJECTS:
-        raise Exception(f"QueryObject fixture not defined for datasource: {query_name}")
-    obj = QUERY_OBJECTS[query_name]
-
-    # apply overrides
-    if ":" in query_name:
-        parent_query_name = query_name.split(":")[0]
-        obj = {
-            **QUERY_OBJECTS[parent_query_name],
-            **obj,
-        }
+        obj = QUERY_OBJECTS[query_name]
```

Comment:
I don't understand the point of this change - if `query_name not in QUERY_OBJECTS`, we probably can't assign `obj = QUERY_OBJECTS[query_name]`? Consequently, the code below becomes unreachable, raising the question if it's needed anymore

Reply:
this is not the updated code